.github/workflows/main.yml

```diff
@@ -42,9 +43,10 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version-file: go.mod
+          check-latest: true
 
       - name: Run tests
-        run: make test
+        run: make test-acc
```

Comment:
We remove the make test (was doing make test-unit and make-test-acc) because make-test-acc will also run unit tests.